scvi/dataset/dataset.py

```diff
@@ -292,7 +292,7 @@ def library_size(X):
         return local_mean, local_var
 
     @staticmethod
-    def get_attributes_from_matrix(X, batch_indices=0, labels=None):
+    def get_attributes_from_matrix(X, batch_indices=0, labels=None, subclusters=None):
```

Comment:
Another example of the bad api of the current GeneExpressiondataset (where you have to change the base class prototype).
Subclusters are an attribute from one of the new dataset, I took the code from jules branch and not sure exactly.
It will be cleaned in #374

Reply:
ok got it